components/brave_wallet_ui/components/extension/transactions-panel/index.tsx

```diff
@@ -34,25 +35,35 @@ import {
 export interface Props {
   selectedNetwork?: BraveWallet.NetworkInfo
   selectedAccountAddress?: string
-  onSelectTransaction: (transaction: SerializableTransactionInfo) => void
+  selectedAccountCoinType?: BraveWallet.CoinType
 }
 
 export const TransactionsPanel = ({
   selectedNetwork,
   selectedAccountAddress,
-  onSelectTransaction
+  selectedAccountCoinType,
 }: Props) => {
   // redux
-  const transactions = useUnsafeWalletSelector(WalletSelectors.transactions)
+  const dispatch = useDispatch()
+
+  // queries
+  const { data: transactionList = [] } = useGetTransactionsQuery({
+    address: selectedAccountAddress ?? null,
+    chainId: null,
+    coinType: selectedAccountCoinType ?? null
+  })
```

Comment:
Does it need a skip arg to avoid pulling all transactions when `selectedAccountAddress` and `selectedAccountCoinType` are empty?

Reply:
updated to skip